doc/api/util.md

```diff
@@ -424,6 +430,9 @@ changes:
     example below. **Default:** `true`.
 * Returns: {string} The representation of passed object
 
+Deprecation notice: The legacy function signature that was changed in v0.9.3
```

Comment:
Nit: remove `Deprecation notice:`.

Reply:
I'd suggest simplifying this greatly:

```markdown
The legacy function signature `util.inspect(object, [showHidden], [depth], [colors])` is deprecated.
```

Reply:
One problem with this is that it can be interpreted as saying that `util.inspect(object)` is deprecated but it is not. Can we indicate that the `showHidden`, `depth`, and `colors` arguments are deprecated instead of pointing to the function signature?